scripts/ai/AIDriveStrategyCombineCourse.lua

```diff
@@ -103,7 +103,6 @@ function AIDriveStrategyCombineCourse:init(task, job)
         }
     })
 
-    -- TODO: move this to a setting
     self.callUnloaderAtFillLevelPercentage = 80

```

Comment:
We can remove this variable and use the setting everywhere instead.

Reply:
I looked after pull request to see if this is really unnecessary, you're right.